.github/workflows/integration-tests.yaml

```
@@ -96,7 +96,7 @@ jobs:
       - name: Run tests
         run: |
           pip install pytest
-          python3 -m pip install -e sdk/python; pytest -s sdk/python/test --log-cli-level=debug --namespace=default
+          python3 -m pip install -e sdk/python[huggingface]; pytest -s sdk/python/test --log-cli-level=debug --namespace=default
```

```suggestion
          python3 -m pip install -e sdk/pythoh/kubeflow/trainer -e sdk/python; pytest -s sdk/python/test --log-cli-level=debug --namespace=default
```

Instead of hard corded depended library, could you use the requirements fo the trainer?
This allows us to avoid double management for dependencies.

---

Thank you for your suggestions! I will update accordingly.